src/StorageTest.tsx

```diff
@@ -31,9 +32,36 @@ export async function storageTest() {
   try {
     await set('idb-test', true);
   } catch (e) {
-    errorLog('storage', 'Failed IndexedDB Test', e);
+    errorLog('storage', 'Failed IndexedDB Set Test - trying to delete database', e);
+    try {
+      await deleteDatabase();
+      await set('idb-test', true);
+      // Report to sentry, I want to know if this ever works
+      reportException('deleting database fixed IDB set', e);
+    } catch (e2) {
+      errorLog('storage', 'Failed IndexedDB Set Test - deleting database did not help', e);
```

Comment:
```suggestion
      errorLog('storage', 'Failed IndexedDB Set Test - deleting database did not help', e2);
```

Reply:
The idea was if the fix didn't work, to throw the original error. But I'm not sure it matters either way.

Reply:
Hm, but this is just error logging and we logged the original error already.